c2cgeoportal/views/layers.py

```diff
@@ -468,15 +468,18 @@ def query_enumerate_attribute_values(dbsession, layerinfos, fieldname):
         return dbsession.query(distinct(attribute)).order_by(attribute).all()
 
 
-def get_layer_class(layer):
-    # exclude the columns used to record the last features update
-    exclude = [] if layer.exclude_properties is None else layer.exclude_properties.split(",")
-    last_update_date = Layers.get_metadata(layer, "lastUpdateDateColumn")
-    if last_update_date is not None:
-        exclude.append(last_update_date)
-    last_update_user = Layers.get_metadata(layer, "lastUpdateUserColumn")
-    if last_update_user is not None:
-        exclude.append(last_update_user)
+def get_layer_class(layer, with_exclude=False):
+    if with_exclude:
+        # Exclude the columns used to record the last features update
+        exclude = [] if layer.exclude_properties is None else layer.exclude_properties.split(",")
```

Comment:
layer.exclude_properties is not part of "the columns used to record the last features update"
It seems that this line shoud be outside of if block.
I suppose we don't need "excluded properties" in select / insert / update operations.

Reply:
@arnaud-morvan I don't understand your comment, as I understand the start say the opposite of the end of your comment.

Reply:
Note that `with_exclude=False` is mostly for insert / update operations, and `with_exclude=True` for metadata operations (Layertree, xsd, lingua)